app/javascript/hooks/forms/users/authentication/useSignInForm.js

```diff
@@ -0,0 +1,68 @@
+import * as yup from 'yup';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useTranslation } from 'react-i18next';
+import { useForm } from 'react-hook-form';
+import { useCallback, useMemo } from 'react';
+
+export function useSignInFormValidation() {
+  return useMemo(() => (yup.object({
+    email: yup.string().required('forms.validations.email.required').email('forms.validations.email.email'),
+    password: yup.string().required('forms.validations.password.required'),
+    extend_session: yup.bool(''),
+  })), []);
+}
+
+export default function useSignInForm({ _defaultValues, ..._config } = {}) {
```

Comment:
That's part of the config merging issue mentioned below.
By default the configuration will be an empty object and the `defaultValues` will be extracted into a separate variable when the rest of the config props will be collected in one level deep object `_config`.